ansible_3par_docker_plugin/tasks/configure_doryd_service.yml

```diff
@@ -14,3 +14,34 @@
       daemon_reload: yes
       enabled: yes
       name: doryd.service
+
+  - name: collect facts about system services
+    service_facts:
+    register: services_state
+
+  - name: If the service failed to start, use another kubeconfig
+    ini_file:
+      path: /etc/systemd/system/doryd.service
+      section: Service
+      option: ExecStart
+      value: /usr/libexec/kubernetes/kubelet-plugins/volume/exec/hpe.com~hpe/doryd /etc/origin/master/admin.kubeconfig  hpe.com
```

Comment:
Stopgap fix. The long term fix would be to understand which config file works for which version of openshift/kubernetes